nbgitpuller/plugins/nbgitpuller-standard/README.md

```diff
@@ -0,0 +1,20 @@
+# nbgitpuller - standard web server download plugin
```

Comment:
Can we name these slightly different? Having the name nbgitpuller doesn't help so much, and having the name standard have made me think quite a while until this README.md was added that this could have been the standard nbgitpuller behavior represented as a plugin.

By having names instead like below, I think such confusion would be far more unlikely.

- downloader-generic-web
- downloader-googledrive
- downloader-dropbox

@manics could you chime in about this suggestion? I'd like some agreement not only from @sean-morris author before asking for this kind of change.

Reply:
Sorry, I'm a bit lost 😄. What are we naming here.... the "friendly" plugin name that a user sees, the folder name, or the Python package name? If the latter I think it should include `nbgitpuller-` as it'll appear in `pip list` so should be informative.

Reply:
I'm thinking about both folder names and pip names, but I think it's acceptable that the folder names exclude a `nbgitpuller-` prefix used in the pip name, but could also include it - perhaps that's most clear then.

Repo name & folder name:

- `nbgitpuller-downloader-generic-web`
- `nbgitpuller-downloader-googledrive`
- `nbgitpuller-downloader-dropbox`

Reply:
My 2 cents is that this functionality is complex and useful enough that we could merge it in, and open a new issue to gather feedback about a potential name change, and then just do that in a separate PR. Otherwise I worry we'd be nesting one important decision (name changes) inside a PR to add a lot of extra functionality.

Reply:
Opened #223 so this doesn't have to be done now!

Reply:
@consideRatio @choldgraf @manics 
ok I think I follow all this. At this point, on this PR. I am going to move the plugins themselves out of the repo and refactor the test files to reflect this. All good.  I struggled to come with a name on the "normal" web plugin! I will go with whatever people think!

Then in #223, we can deal with plugins which would include naming?
And in regards to @manics piece below(https://github.com/jupyterhub/nbgitpuller/pull/194#issuecomment-961193477), we would have a separate PR addressing documentation related to plugin support in nbgitpuller.

Reply:
@sean-morris that is a very acceptable plan. Thank you for being so patient, you are doing some very heavy lifting with this work! Thank you so much for your very thorough effort!!!

Reply:
good deal. I have two questions:
1) I created the plug-in repos in my github account; should they go into the jupyterhub github organization?
2) I built and uploaded the plugins to PyPI also using my account. Is this standard practice or should it live under some other account in PyPI?

Reply:
@sean-morris it's up to you 😃. You can keep it under your account, or you can transfer it to the JupyterHub org. If you want to do the latter the easiest way is to transfer ownership of the repo to a JupyterHub admin who can then transfer it to the organisation.

If you want to transfer it to JupyterHub then PyPI has a jupyterhub-bot account for automated publishing via GitHub workflows: https://pypi.org/user/jupyterhub-bot/

Reply:
@manics I am trying to transfer ownership of the plugin repos and am getting an error message about not being able to create public repositories in jupyterhub
>> You don’t have the permission to create public repositories on jupyterhub

Reply:
@sean-morris That's expected, you can transfer your repo to a JupyterHub _admin_ e.g. @consideRatio (or myself, though I suspect @consideRatio will be faster to respond) we can then transfer it to the JupyterHub org.

Reply:
Great I just made the transfer request to @consideRatio for the three plugin repos.